katas/content/teleportation/prepare_and_send_message/solution.md

```diff
@@ -0,0 +1,11 @@
+The exercise requires to prepare the `qMessage` is an eigenstate of the three Pauli basis.
```

Comment:
Unitaries are associated with the eigenstates, bases - with basis vectors. Let's refer to the states as eigenstates of the corresponding unitary matrix throughout this explanation